specification/compute/resource-manager/Microsoft.Compute/DiagnosticRP/stable/2025-06-01/diagnostic.json

```diff
@@ -0,0 +1,307 @@
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ComputeDiagnosticBase": {
+      "description": "Contains metadata of a diagnostic type",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/DiagnosticProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../../common-types/resource-management/v5/types.json#/definitions/ProxyResource"
+        }
+      ]
+    },
+    "DiagnosticProperties": {
+      "description": "Contains additional properties of a diagnostic",
+      "type": "object",
+      "properties": {
+        "supportedResourceTypes": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "Describes what are the supported resource types for a diagnostic."
+        }
+      }
+    },
+    "ErrorResponse": {
+      "type": "object",
+      "properties": {
+        "error": {
+          "$ref": "../../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorDetail"
+        }
+      },
+      "description": "An error response from the Compute Diagnostic Resource Provider service."
+    },
+    "ErrorDetail": {
+      "type": "object",
+      "properties": {
+        "code": {
+          "type": "string",
+          "description": "The error code."
+        },
+        "target": {
+          "type": "string",
+          "description": "The target of the particular error."
+        },
+        "message": {
+          "type": "string",
+          "description": "User friendly error message."
+        },
+        "details": {
+          "type": "array",
+          "items": {
+            "$ref": "../../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorDetail"
+          },
+          "description": "The Api error details"
+        },
+        "innerError": {
+          "$ref": "#/definitions/InnerError",
+          "description": "The Api inner error"
+        }
+      },
+      "description": "Error Detail message."
+    },
+    "InnerError": {
+      "type": "object",
+      "properties": {
+        "exceptiontype": {
+          "type": "string",
+          "description": "The exception type."
+        },
+        "errordetail": {
+          "type": "string",
+          "description": "The internal error message or exception dump."
+        }
+      },
+      "description": "Inner error details."
+    },
+    "SpotPlacementScoresInput": {
+      "type": "object",
+      "properties": {
+        "desiredLocations": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The desired regions"
+        },
+        "desiredSizes": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ResourceSize"
+          },
+          "description": "The desired resource SKUs."
```

Comment:
>"The desired resource SKUs." [](http://example.com/codeflow?start=25&length=28)

Reconcile property name and description. Also, could you use a more specific term than "resource" (e.g. virtual machine...?).

Reply:
Updated description